docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM frolvlad/alpine-glibc:alpine-3.5
```

Comment:
I agree. But the problem is that alpine uses musl libs and so we have to get some glibc package in there because otherwise siege will just segfault.
Some other potential solution is to use an official alpine image and install https://github.com/sgerrand/alpine-pkg-glibc in the dockerfile but that means we rely on that as a somewhat sketchy other thingy.